### PR TITLE
Saving a slice of frames in a single operation, and removed fsync

### PR DESCRIPTION
**Pull request by ivoras**
See commit descriptions and issue #18 for details.

### CODE REVIEW

**Comment by mattetti** [comment]
Thanks a lot for the PR, I like the direction, I am a bit surprised that a frame is equal to a sample, that doesn't seem right in the case of a file with more than 1 channel (a frame is a sample * num channels).

I need to try to remember why I was syncing when setting up the metadata, it doesn't seem to make a lot of sense but I will try to dig through the history.

Finally, any chance you could add some tests to make sure the code behaves as expected, especially with the type of frames.

**Comment by ivoras** [comment]
It was just a single sample in my case where I have a single channel. I handled the case with multiple channels partially - some checks are needed that the slice length is sane.

The original issue is just a side effect of Go's binary.Write() accepting both simple types (like integers) and slices of those, transparently.

I'll add the tests and submit another PR.

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/go-audio/wav
+module github.com/ivoras/wav
```
Comment (shmsr) [inline go.mod:1]: You shouldn't have included this change with this PR.

Comment (mattetti) [inline go.mod:1]: Should import go-audio/wav

**Review by mattetti — CHANGES_REQUESTED** [review]

**Pull request closed without merging**